### PR TITLE
[source-postgres] CAT test for CDC postgres

### PR DESCRIPTION
**Pull request by xiaohansong**
## What
CAT test for CDC configs

## How
- Add cdc configs
- In setup/teardown we will create replication_slot/publication and clean it.
- [TODO] Added final_teardown logic

## Can this PR be safely reverted and rolled back?

- [ ] YES 💚
- [ ] NO ❌

### CODE REVIEW

File: airbyte-integrations/connectors/source-postgres/integration_tests/seed/hook.py [hunk 1]
```diff
@@ -167,23 +252,26 @@ def delete_schemas_with_prefix(conn: extensions.connection, date_prefix: str) ->
...
+        setup(with_cdc=True)
     elif command == "teardown":
         teardown()
     elif command == "prepare":
         prepare()
+    elif command == "insert":
+        cdc_insert()
     else:
-        ra
+        exit(1)
```
Comment (clnoll) [inline airbyte-integrations/connectors/source-postgres/integration_tests/seed/hook.py:277]:
```suggestion
        print(f"Unrecognized command {command}.")
        exit(1)
```

File: airbyte-integrations/connectors/source-postgres/integration_tests/seed/hook.py [hunk 2]
```diff
@@ -114,39 +126,111 @@ def create_table(conn: extensions.connection, schema_name: str, table_name: str)
...
+        ('4', 'four'),
+        ('5', 'five')
+    ]
+    connection = connect_to_db()
+    table_name = 'id_and_name_cat'
+    if connection:
+        insert_records(connection, schema_name, table_name, new_records)
+        connection.close()
+
+def setup(with_cdc=False):
```
Comment (clnoll) [inline airbyte-integrations/connectors/source-postgres/integration_tests/seed/hook.py:153]: Looks like you can delete the `setup` function on line 137.

File: airbyte-integrations/connectors/source-postgres/integration_tests/seed/hook.py [hunk 3]
```diff
@@ -114,39 +126,111 @@ def create_table(conn: extensions.connection, schema_name: str, table_name: str)
...
-    conn = connect_to_db()
+    # Connect to the database
+    connection = connect_to_db()
 
-    if conn:
-        create_schema(conn, schema_name)
-        create_table(conn, schema_name, table_name)
-        insert_records(conn, schema_name, table_name, records)
-        conn.close()
+    if connection:
```
Comment (clnoll) [inline airbyte-integrations/connectors/source-postgres/integration_tests/seed/hook.py:168]: This `if` statement shouldn't be needed anymore now that `connect_to_db` always returns a value.

**Review by clnoll — APPROVED** [review]
Looks great @xiaohansong, just a few small comments.

**Review by theyueli — APPROVED** [review]
Looks good! (just need to address @clnoll's comments)

File: airbyte-integrations/connectors/source-postgres/integration_tests/seed/hook.py [hunk 4]
```diff
@@ -114,39 +126,111 @@ def create_table(conn: extensions.connection, schema_name: str, table_name: str)
...
+    schema_name = load_schema_name_from_catalog()
+    new_records = [
+        ('4', 'four'),
+        ('5', 'five')
+    ]
+    connection = connect_to_db()
+    table_name = 'id_and_name_cat'
+    if connection:
+        insert_records(connection, schema_name, table_name, new_records)
+        connection.close()
```
[the remark this answers is not in the document]
Comment (xiaohansong) [reply airbyte-integrations/connectors/source-postgres/integration_tests/seed/hook.py:151]: done

**Pull request merged**